project-base/src/Shopsys/ShopBundle/DataFixtures/Demo/PricingGroupDataFixture.php

```diff
@@ -6,16 +6,20 @@
 
 use Doctrine\Common\Persistence\ObjectManager;
 use Shopsys\FrameworkBundle\Component\DataFixture\AbstractReferenceFixture;
+use Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig;
 use Shopsys\FrameworkBundle\Component\Domain\Domain;
 use Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroupData;
 use Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroupDataFactoryInterface;
 use Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroupFacade;
+use Shopsys\FrameworkBundle\Model\Pricing\Group\PricingGroupSettingFacade;
 
 class PricingGroupDataFixture extends AbstractReferenceFixture
 {
+    public const PRICING_GROUP_ORDINARY_DOMAIN = 'pricing_group_ordinary_domain';
+    public const PRICING_GROUP_PARTNER_DOMAIN = 'pricing_group_partner_domain';
+    public const PRICING_GROUP_VIP_DOMAIN = 'pricing_group_vip_domain';
+
     public const PRICING_GROUP_ORDINARY_DOMAIN_1 = 'pricing_group_ordinary_domain_1';
```

Comment:
what is "this" referencing to? All these constants are used somewhere, except of PRICING_GROUP_VIP_DOMAIN - removed in bacb4b8a9

Reply:
i'm referencing to `PRICING_GROUP_ORDINARY_DOMAIN_1` because I don't see it being used in this file and I don't think that it should be used at all.. you should use the `PRICING_GROUP_ORDINARY_DOMAIN` for domain 1

Reply:
should be resolved by 10cec4d